tests/analysis/test_gain_loss.py

```
@@ -254,3 +255,50 @@ def test_plot_returns_figure_from_gainloss(sample_df):
 
     fig = gl.plot()
     assert isinstance(fig, Axes)
+
+
+def test_with_custom_column_names(sample_df):
+    """Test GainLoss with custom column names."""
+
+    rename_mapping = {
+        get_option("column.customer_id"): custom_columns["column.customer_id"],
```

Ruff (0.11.9)

Undefined name `custom_columns`

(F821)